spec/student_spec.rb

```diff
@@ -0,0 +1,13 @@
+require './student'
+
+describe Student do
+  context 'Creating a student' do
+    it "Is the student's information correct?" do
+      s = Student.new(21)
+
+      s.name = 'May'
+      expect(s.age).to eq(21)
+      expect(s.name).to eq('May')
+    end
+  end
+end
```

Comment:
- [ ] There is a test case missing for `play_hooky` class in student class. Please make sure to create a test block for `play_hooky` method as well.